doc/source/_static/custom.css

```diff
@@ -1,16 +1,61 @@
 .sphinx-codeautolink-a{
-    border-bottom-color: rgb(0, 0, 0);
-    border-bottom-style: dotted;
-    border-bottom-width: 1px;
+  border-bottom-color: rgb(0, 0, 0);
+  border-bottom-style: dotted;
+  border-bottom-width: 1px;
 }
 .sphinx-codeautolink-a:hover{
-    color: rgb(255, 139, 139);
+  color: rgb(255, 139, 139);
 }
 
 .supportbutton a {
- color: var(--pst-color-sidebar-link-active);
+color: var(--pst-color-sidebar-link-active);
 }
 
 .navbar-brand {
-  display: inline-block !important;
+display: inline-block !important;
+}
+
+
+/* -------------------- EXAMPLE GALLERY OVERRIDES -------------------- */
+
+.example-gallery {
```

Comment:
Add comments as for what each blog it does, even if it seems obvious. The overlap betweeen bayesian expert and css experts is quite small